docs/next/guides/building-and-testing/file-structure.md

```diff
@@ -13,20 +13,57 @@ tags:
 
 # File structure
 
-The source files for this project are stored on GitHub in a [mono repository](https://github.com/handsontable/handsontable).
-```
-├── bin                    # Binary files
-├── dist                   # Compiled files
-├── docs                   # Documentation files
-├── examples               # Code examples
-├── handsontable.d.ts      # TypeScript definitions file
-├── languages              # Translations (i18n)
-├── resources              # Static files for README.md
-├── scripts                # Scripts for building and deploying
-├── src                    # Source files
-├── test                   # Automated tests
-└── wrappers
-    ├── angular            # Wrapper for Angular
-    ├── react              # Wrapper for React
-    └── vue                # Wrapper for Vue 2
-```
+Handsontable's source files are stored on GitHub, in a [mono repository](https://github.com/handsontable/handsontable).
```

Comment:
I'm wondering if the structure isn't too detailed? WDYT @wojciechczerniak?
On one hand, it explains the purpose of everything, but on the other it looks quite overwhelming, I don't know.

Reply:
It is.

What is the question we want to answer here? IMO we want to show where packages / wrappers / tests and source code can be found. IMO `.nvmrc`, `.gitignore` etc. are irrelevant here. May change anytime. And if someone want to see such detailed structure, can open the repo.

Reply:
@jansiegel, @wojciechczerniak - ok, simplifying the structure: 38ea009